### DISCUSSION TITLE
I cannot login to vercel

### DISCUSSION THREAD

**Discussion by jaykim19** [vercel/community#3409] [Account Issue] [answered]
### Question

I created the account with Github account.
I entered the phone number to create an account, but I cannot login because the phone number is already linked with.
I tried to another email addresses, but I had to enter the phone number, so I cannot login at all.
I don't know how to do it.

**Comment by amyegan** [comment, marked as answer]
Hey @jaykim19. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. That will let you verify account details privately so the team can help you get logged in.

**Reply by UgochukwuO** [reply]
Hello. I'm having the same issue. An email has been sent the email to register@vercel.com 3 buisness days ago. Is this wait time normal? Might the email have been missed? Should one send another? Onto another related topic, supposing we were running with Vercel in production and had account issue like this, should we expect to not hear back within hours from the support team? Or is is generally practice to not run Vercel in production? I ask because one would assume that support responses would be timely due to the nature of a potential paying customer's infrastructure being activity being blocked until the problem was resolved. Thanks for your time in reading this, hopefully I posed the questions well and someone can help me learn how one should go forward from here.

**Reply by Pieparker** [reply]
Hey there! Please ensure that you have emailed the correct inbox. Additionally, registration emails are handled in the order that they are received and the team strive to respond as soon as possible.
